docs/manual-EN/1.overview/2.quick-start/1.get-started.md

```diff
@@ -1,306 +1,452 @@
------------------------------
-
--- By temporary variable
-nebula> $var=GO FROM 201 OVER follow yield follow._dst as id; \
-  GO FROM $var.id OVER choose YIELD $^.student.name AS Student,\
-  $$.course.name AS Course, choose.grade AS Grade;
-=============================
-| Student |  Course | Grade |
-=============================
-|  Monica |    Math |     6 |
------------------------------
-|  Monica | English |     3 |
------------------------------
-|    Jane | English |     3 |
------------------------------
-```
-
-* `|` denotes a pipe. The output of the formal query acts as input to the next one follow a pipeline.
-* `$-` refers to the input stream.
+GO FROM 100 OVER follow YIELD follow._dst AS id | \
+GO FROM $-.id OVER serve YIELD $$.team.name AS Team;
+```
+
+The following information is returned.
+
+```
+===========
+| Team    |
+===========
+| Nuggets |
+-----------
+```
+
+Enter the following statement with a `temporary variable`:
+
+```ngql
+$var=GO FROM 100 OVER follow YIELD follow._dst AS id; \
+GO FROM $var.id OVER serve YIELD $$.team.name AS Team;
+```
 
-The second approach adopts a user-defined variable `$var`. The scope of this variable is within the compound statement.
+The following information is returned.
 
-## Tips
+```
+===========
+| Team    |
+===========
+| Nuggets |
+-----------
+```
 
-In practice, it often takes several retries to find a proper graph schema during testing. You can put all the DDL statements in a file as follows.
+**Note**:
+
+* `|` denotes a pipe. The output of the previous query acts as ab input to the next query.
+* `$-` refers to the input stream.
+* The second approach adopts a user-defined variable `$var`. The scope of this variable is within the compound statement.
+
+## Batch Inserting
+
+To insert multiple data, you can put all the DDL (data definition language) statements in a `.ngql` file as follows.
 
 ```ngql
-DROP SPACE test; -- Delete useless space
-CREATE SPACE test(partition_num=1024, replica_factor=1);
-USE test;
-CREATE TAG course(name string, credits int);
-CREATE TAG building(name string);
-CREATE TAG student(name string, age int, gender string);
-CREATE EDGE like(likeness double);
-CREATE EDGE select(grade int);
+CREATE SPACE nba(partition_num=1024, replica_factor=1);
+USE nba;
+CREATE TAG player(name string, age int);
+CREATE TAG team(name string);
+CREATE EDGE follow(degree int);
+CREATE EDGE serve(start_year int, end_year int);
 ```
 
-Then batch write to console
+* If you install **Nebula Graph** by compiling the source code, you can batch write to console by the following command:
 
-```bash
+```$
 cat  schema.ngql | ./bin/nebula -u user -p password
 ```
 
-Likewise, you can put hundreds or thousands DML statements in a data.ngql to insert. If you have millions of records to insert, please use the csv importer tool (or sst ingest tool).
+* f you are using **Nebula Graph** by docker-compose, you can batch write to console by the following command:
+
+```$
```

Comment:
$ >- bash
let's unify the format here

Reply:
@whitewum @Amber1990Zhang I am confused about when to use bash or $ or other signs.